merge_files.py

```diff
@@ -4,11 +4,22 @@
 
 @author: GRENTOR
 """
+import logging
 import json
 import os
 from jsonmerge import Merger
 from schema_modified import schema_builder
 
+logger = logging.getLogger("jsonmerge-logger")
+logger.setLevel(level=logging.INFO)
+if not logger.handlers:
+    # Prevent logging from propagating to the root logger
+    logger.propagate = 0
+    log_formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    stream_handler = logging.StreamHandler()
+    stream_handler.setFormatter(log_formatter)
+    logger.addHandler(stream_handler)
+

```

Comment:
Can you move this to jsonmerge_utils.py and put it in a function like format_logger() or something , 
all the settings/formatting of logger object should be in the main module (jsonmerge_utils.py).

Reply:
consider it done! Ill send a new PR within the hour.

Reply:
You dont have to create a new PR each time, just make changes in your local repo, commit them, and push the commits
The changes are automatically reflected in PR

Reply:
I had an interesing repository that forced me to squash the commits into one. Since I was not familiar with git I had troubles giving them what they wanted :(. Thank you for reminding me that I can just commit :).

Reply:
ah yes, we have to squash commits into one, but that can be done from my end as well (recently found out) so no worries there
but in case you are curious, you can refer to this [article](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec)